news/resample-relocation.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Moved resampler out of parsers, new path is diffpy.utils.resample
```

Comment:
@alisnwu I think we are using reampler.py?

Reply:
Will update the news file